Jenkinsfile

```diff
@@ -97,6 +97,59 @@ pipeline {
             }
           }
         }
+        // We have 2 stages for GCE Authenticator tests. The first one runs inside
+        // a GCE instance and retrieves all the tokens that will be used in the tests.
+        // It then stashes the tokens, which are unstashed in the stage that runs the
+        // GCE Authenticator tests using the tokens.
+        // This way we can have a light-weight GCE instance that has no need for conjurops
+        // or git identities and is not open for SSH
+        stage('GCE Authenticator preparation - Allocate GCE Instance') {
+          steps {
+            script {
+              node('executor-v2-gcp-small') {
+                sh '''
+                  get_token_into_file() {
+                    local token_format="$1"
+                    local audience="$2"
+                    local filename="$3"
+
+                    curl \
+                      -s \
+                      -H 'Metadata-Flavor: Google' \
+                      "http://metadata/computeMetadata/v1/instance/service-accounts/default/identity?format=${token_format}&audience=${audience}" \
```

Comment:
I think we should change `default` to the name of the service account, so if something will be changed in GCE configuration we will fail here and not later in the tests

Reply:
@nessiLahav can you elaborate? what is the difference between running this request with `default` and running it with the service-account name?

how will a change in GCE config will break this?

i am not keen on using the service account name as we don't have it here (as far as i know) and we will need to store it in `conjurops`, which is something that we are trying to avoid.

Reply:
is service account name the same as service-account id?

Reply:
@eranha ^

Reply:
An example is for the following service account: `nessi-service-account-test@refreshing-mark-284016.iam.gserviceaccount.com`
`curl --header "Metadata-Flavor: Google" --data-urlencode "audience=conjur/cucumber/host/nessitest/hosttest1" --get --data-urlencode "format=full" "http://metadata/computeMetadata/v1/instance/service-accounts/nessi-service-account-test@refreshing-mark-284016.iam.gserviceaccount.com/identity"`

Since we are relying on this service account id later in the resource restrictions tests, i prefer to found out here that something change, instead of later in the tests.

Reply:
oh you're using the whole id. got it.

anyway - i think we need a good reason to use this and not `default` as the service account email will need to be saved in conjurops. this requires the instance to be conjurized and it will need to be changed in case the SA is replaced.

so unless we have a good reason to use it, i would stay with default.

what do you think?

Reply:
OK, this is too much effort, i agree to stay with the default